UnityProject/Assets/Scripts/Player/PlayerNetworkActions.cs

```diff
@@ -281,6 +281,32 @@ public void CmdDropAllItems(uint itemSlotID, Vector3 Target)
 		}
 	}
 
+	/// <summary>
+	/// Transfers x amount of items from one hand to another. For stackable items only
+	/// </summary>
+	[Command]
+	public void CmdSplitStack(uint fromSlotID, uint toSlotID, NamedSlot fromSlot, NamedSlot toSlot, int amountToTransfer)
+	{
+		if (fromSlot != NamedSlot.leftHand && fromSlot != NamedSlot.rightHand || toSlot != NamedSlot.leftHand && toSlot != NamedSlot.rightHand) return; //Only allowed to transfer from one hand to another
+		if (!Validations.CanInteract(playerScript, NetworkSide.Server, allowCuffed: false)) return; //Not allowed to transfer while cuffed
+		if (!Cooldowns.TryStartServer(playerScript, CommonCooldowns.Instance.Interaction)) return;
+		if (NetworkIdentity.spawned.ContainsKey(fromSlotID) == false) return;
+
+		var ObjFS = NetworkIdentity.spawned[fromSlotID].gameObject;
+		var ObjTS = NetworkIdentity.spawned[toSlotID].gameObject;
```

Comment:
Just need to check to make sure that player doing action is observer of the storage

Reply:
Fixed in new commit